spec/xcpretty/parser_spec.rb

```diff
@@ -669,5 +669,13 @@ def given_kiwi_tests_are_done
       end
     end
 
+    context "performance" do
```

Comment:
Consider adding this as integration (cucumber) test and not unit.

Reply:
I don't see a reason to do this, let's discuss 2f2. (I actually implemented this, it was annoying).